File: Latihan B/Java/Processor.java
public class Processor{

	//atribut private
	private String name; //nama prosesor
	private int price; //harga prosesor


	//konstruktor
	public Processor(){

	}

	public Processor(String name, int price){
		this.name = name;
		this.price = price;
	}

	//getter dan setter atribut name
	public void setName(String name){
		this.name = name;
	}

	public String getName(){
		return this.name;
	}

	//getter dan setter atribut price
	public void setPrice(int price){
		this.price = price;
	}

	public int getPrice(){
		return this.price;
	}

}

File: Latihan B/PHP/Processor.php
<?php

class Processor{

	//atribut private
	private $name = ""; //nama prosesor
	private $price = 0; //harga prosesor


	//konstruktor
	public function __construct($name = "", $price = 0){
		$this->name = $name;
		$this->price = $price;
	}

	//getter dan setter atribut name
	public function setName($name){
		$this->name = $name;
	}

	public function getName(){
		return $this->name;
	}

	//getter dan setter atribut name
	public function setPrice($price){
		$this->price = $price;
	}

	public function getPrice(){
		return $this->price;
	}

}

?>